developer-guide/index.md

```diff
@@ -288,15 +319,17 @@ The payload **MUST** contain the following properties:
 
 The payload **CAN** have the following properties:
 
-* `userInput`: as documented above
-* `submitted`: as documented above
+* `userInput` _object_: as documented above
+* `submitted` _bool_: as documented above
 
 ### Securing your data
 
 At GetFeedback we value the privacy and security of your data, for this reason, we do not allow unauthenticated requests to our feedback collection services.
 
 In order to submit feedback items via HTTP requests, you always need to provide an access token. The procedure to request an access token is described in the following sections.
 
+> **Note:** while we require authentication for submitting feedback items, we don't enforce it upon form configuration retrieval.
```

Comment:
Don't think that `enforce` is the correct wording here since it's simply not implemented?

Reply:
Yep, changed with

> **Note:** while we enforce authentication for submitting feedback items, we don't require it upon form configuration retrieval.